### PR TITLE
Implement IElevator Wrapper 'ElevatorController'

### PR DESCRIPTION
**Pull request by AnitaMichlits**

### CODE REVIEW

**Review by MikeEdu — CHANGES_REQUESTED** [review]

File: src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+            throw(new RuntimeException("Error in setServicesFloors: " + ex.getMessage()));
+        }
+    }
+
+    public void setTarget(int elevatorNumber, int target) {
+        try{
+            elevatorCenter.setTarget(elevatorNumber, target);
+        }
+        catch(RemoteException ex){
+            throw(new RuntimeException("Error in setTarget: " + ex.getMessage()));
```
Comment (MikeEdu) [inline src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java:234]: Test exception throwing of setter methods.

File: src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java [hunk 2]
```diff
@@ -0,0 +1,237 @@
+package at.fhhagenberg.sqe.ecc;
+import sqelevator.IElevator;
+
+import java.rmi.RemoteException;
+
+
+public class ElevatorController implements IElevatorController {
```
Comment (MikeEdu) [inline src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java:7]: Rename `ElevatorController`

File: src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+        try {
+            switch (elevatorCenter.getCommittedDirection(elevatorNumber)){
+                case IElevator.ELEVATOR_DIRECTION_UP:
+                    dir = CommittedDirection.UP;
+                    break;
+                case IElevator.ELEVATOR_DIRECTION_DOWN:
+                    dir = CommittedDirection.DOWN;
+                    break;
+                default:
+                    dir = CommittedDirection.UNCOMMITED;
```
Comment (MikeEdu) [inline src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java:28]: Please, fix my type UNCOMMITED to UNCOMMITTED. :P

File: src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java (hunk 2 above)
Comment (AnitaMichlits) [reply src/main/java/at/fhhagenberg/sqe/ecc/ElevatorController.java:7]: To what should I rename it? any ideas?

**Review by MikeEdu — APPROVED** [review]

**Pull request merged**